Services/AccessControl/interfaces/interface.ilRBACAccessHandler.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/**
+ * Class ilAccessHandler
+ *
+ * Checks access for ILIAS objects
+ *
+ * @author  Alex Killing <alex.killing@gmx.de>
+ * @author  Sascha Hofmann <saschahofmann@gmx.de>
+ * @version $Id$
+ *
+ * @ingroup ServicesAccessControl
+ */
+interface ilRBACAccessHandler {
```

Comment:
Thanks for adding this interface!
I think the number methods inside can be reduced in the future. The access level of most methods is private. 
I will grep for method calls and change the acces level later.